pkg/core/runtime/component/component.go

```diff
@@ -122,7 +122,6 @@ func (cm *manager) startLeaderComponents(stop <-chan struct{}, errCh chan error)
 			closeLeaderCh()
 		},
 	})
-	log.Info("Starting leader election")
```

Comment:
Moved this log to Postgres implementation. Otherwise on Kuma DP logs was `starting leader election`